node/grpc/server_v2_test.go

```diff
@@ -84,6 +84,23 @@ func TestV2NodeInfoRequest(t *testing.T) {
 	assert.True(t, err == nil)
 }
 
+func TestV2ServerWithoutV2(t *testing.T) {
+	config := makeConfig(t)
+	config.EnableV2 = false
+	c := newTestComponents(t, config)
+	_, err := c.server.StoreChunks(context.Background(), &pbv2.StoreChunksRequest{})
+	require.Error(t, err)
```

Comment:
These block of code may be benefited from a small helper function (parse error and assert the error status), as they repeat many times in this file

Reply:
Done